README.rst

```diff
@@ -45,102 +45,46 @@ Catalog Contents
-a new version is released.
-
-To set up a GCP billing project and use it for authentication when accessing the
-catalog:
-
-* `Create a project on GCP <https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project>`__;
-  if this is the first time using GCP, a prompt should appear asking you to choose which
-  Google account to use for your GCP-related activities. (You should also receive $300
-  in initial cloud credits.
-* `Create a Cloud Billing account <https://cloud.google.com/billing/docs/how-to/manage-billing-account#create_a_new_billing_account>`__
-  associated with the project and `enable billing for the project
-  <https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project>`__
-  through this account.
-* `Using Google Cloud IAM <https://cloud.google.com/iam/docs/granting-changing-revoking-access#granting-console>`__,
-  add the **Service Usage Consumer** role to your account, which enables it to make
-  billed requests on the behalf of the project.
-* Install the `gcloud utilities <https://cloud.google.com/sdk/docs/install>`__ on your
-  computer. This can be done using ``conda`` (or ``mamba``):
-
-.. code:: bash
-
-  conda install -c conda-forge google-cloud-sdk
-
-* Initialize the ``gcloud`` command line interface, logging into the account used to
-  create the aforementioned project and selecting it as the default project; this will
-  allow the project to be used for requester pays access through the command line:
-
-.. code:: bash
-
-  gcloud auth login
-  gcloud init
-
-* Finally, use ``gcloud`` to establish application default credentials; this will allow
-  the project to be used for requester pays access through applications:
-
-.. code:: bash
+See also:
 
-  gcloud auth application-default login
+* `pudl-catalog issues <https://github.com/catalyst-cooperative/pudl-catalog/issues>`__
+* `intake-sqlite issues <https://github.com/catalyst-cooperative/pudl-catalog/issues>`__
 
-* To test whether your GCP account is set up correctly and authenticated you can run the
-  following command to list the contents of the cloud storage bucket containing the
-  intake catalog data:
+PUDL Catalog Usage
+------------------
 
-.. code:: bash
+Accessing Public Cloud Data with Requester Pays
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   gsutil ls gs://intake.catalyst.coop
+To control the cost of distributing potentially large quantities of public data, we are
+using "requester pays." This means that whoever is downloading the data pays the modest
+data egress fees. If you're not familiar with GCP or requester pays, we've written a
+short guide to setting up a GCP project to work with this type of data. See the
+`PUDL Catalog documentation <https://catalystcoop-pudl-catalog.readthedocs.io/en/latest/>`__.
+for details, and also the
+`GCP Documentation on accessing Requster Pays data <https://cloud.google.com/storage/docs/using-requester-pays#using>`__
```

Comment:
little typo that was bugging me: "Requster" --> "Requester"

Reply:
Hah, whoops. Funny that I did this repeatedly.